app/controllers/concerns/mentions.rb

```diff
@@ -0,0 +1,9 @@
+module Mentions
```

Comment:
Not convinced about the name convention of our controller concerns, it's all over the place:

- include ActivityTracking
- include ContentFromTemplate
  include ContentFromTemplate
  include ConflictResolver
  include ConflictResolver
  include Mentions
  include MultipleDestroy
  include MultipleDestroy
  include NotificationsReader
  include NotificationsReader
  include ProjectScoped
  include ProjectScoped

Reply:
noted